modules/products/bitbucket/variables.tf

```diff
@@ -136,5 +136,9 @@ variable "elasticsearch_replicas" {
 variable "display_name" {
   description = "The display name of Bitbucket instance."
   type        = string
-  default     = null
+  default     = ""
+  validation {
+    condition     = length(var.display_name) >= 0 && length(var.display_name) < 255
```

Comment:
This will work with `null` as default:

```
default     = ""
  validation {
    condition     = var.display_name == null || can(regex("^([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$", var.display_name))
    error_message = "Bitbucket display name must be a non-empty value less than 255 characters."
  }
```

Reply:
You're absolutely right! @nghazali 
`can()` and `regex()` combo is a powerful tool, given that regex() throws error when not matching. 
https://www.terraform.io/language/functions/can 
https://www.terraform.io/language/functions/regex 

I've modified the code, if you're happy about the change, do you mind helping merge this PR on Monday?

Reply:
@yzha645 , the modification you made in regx is brilliant. Well done! 🚀